tests/python/gaia-ui-tests/requirements.txt

```diff
@@ -1,13 +1,5 @@
 boto>=2.32.1
-marionette_driver==0.9
-marionette_client==0.16
-mozlog==2.11
-mozcrash==0.14
-moznetwork==0.26
-mozprofile==0.24
-mozrunner==6.7
-mozversion==1.2
-mozdevice==0.45
-moztest==0.7
+marionette_driver>=0.9
```

Comment:
We just got hit by this issue again, see https://bugzilla.mozilla.org/show_bug.cgi?id=1196489#c2
It's rather disruptive to have all Jenkins jobs fail because some change in marionette went in that our Gaia UI test can't handle, because it's out of date.